### PR TITLE
added sample data support to show sample data on dataset details page.

### LINKED ISSUE #125: Add sample data support
Ingestion to ingest sample data to the tables
UI to show sample data in data asset details page

### PR DESCRIPTION
**Pull request by Sachin-chaurasiya**
Closes #125 
### Describe your changes :

I worked on the dataset details page because of the need to show sample data.

#
### Type of change :

- [x] New feature

#
### Frontend Preview (Screenshots) :
image

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/open-source-community/developer) document.
- [x] I have performed a self-review of my own. 
- [x] I have tagged my reviewers below.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [x] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#
### Reviewers
@shahsank3t, @darth-coder00

### CODE REVIEW

**Review by shahsank3t — CHANGES_REQUESTED** [review]
@Sachin-chaurasiya 
Column data type colors should be consistent so let's remove that and keep consistent as its visible in schema tab.

**Comment by Sachin-chaurasiya** [comment]
> @Sachin-chaurasiya
> Column data type colors should be consistent so let's remove that and keep consistent as its visible in schema tab.

sure.

**Comment by Sachin-chaurasiya** [comment]
image

**Review by shahsank3t — APPROVED** [review]

**Comment by shahsank3t** [comment]
@darth-coder00 Please review when you get a chance.

File: catalog-rest-service/src/main/resources/ui/src/components/my-data-details/SchemaTable.tsx [hunk 1]
```diff
@@ -231,177 +231,182 @@ const SchemaTable: FunctionComponent<Props> = ({
...
-                )}
-                data-testid="column"
-                id={column.name}
-                key={index}
-                ref={rowRef}>
-                <td className="tw-relative tableBody-cell">
-                  {getConstraintIcon(column.columnConstraint)}
-                  <span>{column.name}</span>
-                </td>
+      <div className="tw-table-responsive">
```
Comment (darth-coder00) [inline catalog-rest-service/src/main/resources/ui/src/components/my-data-details/SchemaTable.tsx:234]: Why did we make schema table responsive?

Comment (Sachin-chaurasiya) [reply catalog-rest-service/src/main/resources/ui/src/components/my-data-details/SchemaTable.tsx:234]: because it was going outside the viewport.

**Pull request merged**